app/views/trainees/check_details/show.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <h1 class="govuk-heading-l">
   <span class="govuk-caption-l">
-    Draft record for ...
+    Draft record for <%= trainee_name(@trainee).titlecase  %>
```

Comment:
We're not using `titlecase` anywhere else in the code when we render people's names. Let's just assume they've inputted them nicely. One reason for this is my own name!

```rb
"Abigail McPhillips".titleize
=> "Abigail Mc Phillips"
```

:(

Reply:
good point 😅